tests/test_taar_lite_guidranking.py

```diff
@@ -0,0 +1,64 @@
+"""Test suite for taar_lite_guidranking Job."""
+
+import boto3
+from moto import mock_s3
+from mozetl.taar import taar_lite_guidranking
+from pyspark.sql import Row
+
+"""
+Expected schema of co-installation counts dict.
+| -- key_addon: string(nullable=true)
+| -- coinstallation_counts: array(nullable=true)
+| | -- element: struct(containsNull=true)
+| | | -- id: string(nullable=true)
+| | | -- n: long(nullable=true)
+"""
+
+
+MOCK_TELEMETRY_SAMPLE = [
+    Row(addon_guid="test-guid-1", install_count=100),
+    Row(addon_guid="test-guid-2", install_count=200),
+    Row(addon_guid="test-guid-3", install_count=300),
+    Row(addon_guid="test-guid-4", install_count=400),
+]
+
+EXPECTED_ADDON_INSTALLATIONS = {u'test-guid-1': 100,
+                                u'test-guid-2': 200,
+                                u'test-guid-3': 300,
+                                u'test-guid-4': 400}
+
```

Comment:
I would recommend including an "integration" test, using both `@mock_s3`, as well as some fake spark data loaded into the `SparkSession` using `dataframe.createOrReplaceTempView`. Then we can have a test of `extract_telemetry` as well.

Reply:
@fbertsch thanks!  I added a createOrReplaceTempView call in the test suite to exercise the extract_telemetry call.